PasswordRules.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace PasswordChecker
+{
+    public class PasswordRules
+    {
+        public bool isValid = false;
+        private bool ContainsUpperLetter(string pass)
+        {
+            foreach (char c in pass)
+            {
+                if ((Char.IsLetter(c)) && (Char.IsUpper(c)))

```

Comment:
```suggestion
                if (char.IsLetter(c) && char.IsUpper(c))
```

One pharentesis is redundant here. Compiler has a pretty nice feature - if you use `&&` then it works like this:
1. Checks if the first condition evaluates to `true`, if it does not, it skips the remaining part.

A small example:
`if (false && i > 10)` will always evaluate to `false` and it will never check if `i` is actually greater than `10`. :)

Also C# has aliases for class names such as  `Char`, `Int32`, `String`. Just like in Program.cs you did `string.Empty`, here you can do `char.IsLetter()`. It is not a mistake though, but I believe it's common practice in C# to use the alias in such cases :)